templates/output.html.twig

```diff
@@ -39,28 +39,48 @@
         {% include '_tesseract_options.html.twig' with {engine: engine} %}
 
         <div class="form-group submit-btn">
-            <input type="hidden" name="crop[x]" value="{% if crop.x is defined %}{{ crop.x }}{% endif %}" />
-            <input type="hidden" name="crop[y]" value="{% if crop.y is defined %}{{ crop.y }}{% endif %}" />
-            <input type="hidden" name="crop[width]" value="{% if crop.width is defined %}{{ crop.width }}{% endif %}" />
-            <input type="hidden" name="crop[height]" value="{% if crop.height is defined %}{{ crop.height }}{% endif %}" />
             <input type="submit" value="{{ msg( 'submit' ) }}" class="btn btn-info" />
         </div>
-    </form>
 
-    {% if text is defined %}
-        <div class="ocr-output">
-            <hr/>
-            <div class="row">
-                <div class="col-md-6">
-                    <div><!-- This div is required for Cropper.js -->
-                        <img id="source-image" class="img-responsive" src="{{ app.request.get('image') }}" alt="{{ msg('img-alt-text') }}" />
+        {% if text is defined %}
+            <div class="ocr-output">
+                <hr/>
+                <p>{{ msg('drag-help') }}</p>
+                <div class="row">
+                    <div class="col-md-6">
+                        <div class="buttons">
+                            <div class="btn-group" role="group">
+                                <button type="button" class="btn btn-default active drag-mode drag-mode-crop"
+                                        title="{{ msg('drag-mode-crop') }}" data-drag-mode="crop" disabled>
+                                    <img src="{{ asset('build/images/Crop_-_The_Noun_Project.svg') }}" height="20" width="20" />
```

Comment:
Should this have an `alt` attribute (e.g. `alt="Crop"`) for cross-device compat? Unsure how that would work inside a button, though.